### PR TITLE
Re-generate protos for Ruby

### PR DESCRIPTION
**Pull request by stanley-cheung**
Similar to #24880 

Re-generate protos for Ruby ahead of the work needed to modify the xDS client. We need these new classes to be generated.

This PR basically ran the following commands:

```
$ ./tools/bazel build @com_google_protobuf//:protoc
$ ./tools/bazel build src/compiler:grpc_ruby_plugin
$ ./src/ruby/pb/generate_proto_ruby.sh
```

### CODE REVIEW

File: src/ruby/pb/generate_proto_ruby.sh [hunk 1]
```diff
@@ -40,7 +42,13 @@ $PROTOC -I . \
     --plugin=$PLUGIN
 
 $PROTOC -I . \
-    src/proto/grpc/testing/{messages,payloads,stats,benchmark_service,report_qps_scenario_service,worker_service,control}.proto \
+    src/proto/grpc/testing/messages.proto \
```
Comment (jtattermusch) [inline src/ruby/pb/generate_proto_ruby.sh:45]: any reason why this change or is this just refactoring?

**Review by jtattermusch — APPROVED** [review]
LGTM with a question, but I'd like @apolcyn  to approve as well.

**Review by apolcyn — APPROVED** [review]

File: src/ruby/pb/generate_proto_ruby.sh (hunk 1 above)
Comment (stanley-cheung) [reply src/ruby/pb/generate_proto_ruby.sh]: For some reason, that syntax (`{messages,test,empty}.proto`) sometimes doesn't work for me. Ran into some sort of syntax error when I run the original script unmodified. But I couldn't reproduce that anymore. I have reverted this change.
